### DISCUSSION TITLE
Skylanders Imaginators

### DISCUSSION THREAD

**Discussion by mydemonguy** [skyline-emu/skyline#676] [Game Requests]
Skylanders Imaginators

**Comment by MysticAx0lotl** [comment]
Unfortunately, due to the nature of the game (that being toys-to-life), I'm fairly certain Imaginators (or any other Skylanders game) will never work on Skyline. Few emulators period support the feature, with Dolphin (the desktop version) and RPCS3 the only ones I can think of that do

**Reply by PixelyIon** [reply]
It would be a lot of effort to add support for a single game, Amiibos would also be the priority in any case.

**Reply by MysticAx0lotl** [reply]
Agreed. Dolphin is only compatible with TTL games because of its support for USB passthrough. An approach like that on Android would be cumbersome at best.

**Comment by mydemonguy** [comment]
ya just did not want this game to be forgotten i know that this has a save file that holds all the Skylander was not look for support for the portal or anything like that on this one
